sdk/PrebidServerAdapter/PrebidServerModule/PBServerRequestBuilder.m

```diff
@@ -87,7 +87,7 @@ - (NSDictionary *)openrtbRequestExtension: (NSString *)accountId
     NSMutableDictionary *requestPrebidExt = [[NSMutableDictionary alloc] init];
     requestPrebidExt[@"targeting"] = @{};
     requestPrebidExt[@"storedrequest"] = @{@"id" :accountId};
```

Comment:
Looks like this enables server cache for both DFP and Mopub is it the intended behavior??